### PR TITLE
Clonning Website Reservasi Command Center

### PR DESCRIPTION
**Pull request by angaihimitsu**
duplicate website Reservasi Command Center to Reservasi Museum Gedung Sate

### CODE REVIEW

**Review by yoslie — COMMENTED** [review]
same issue with microsite proj., I suggest to use .svg format file rather than .png #cmiiw

and, why the image have large empty space in y axis?

File: components/Menu.vue [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <template>
   <div>
     <div class="flex justify-center">
-      <img width="48" height="48" src="~assets/icon/icon-digiteam.svg" alt="digiteam">
-      <h1 class="text-green-700 text-2xl leading-14 self-center font-bold font-sans">
+      <img width="48" height="48" src="/logomgs.png" alt="digiteam">
```
Comment (doohanas) [inline components/Menu.vue:4]: Hello @angaihimitsu, why did you change this image?

Comment (angaihimitsu) [reply components/Menu.vue:4]: Sorry sir, I think it should be changed completely 😅
